bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,978 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="ehub-wloadconsq-l1">
+		<item-type>Number:Energy</item-type>
+		<label>EHUB, WLoadConsq L1</label>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="ehub-wloadconsq-l2">
+		<item-type>Number:Energy</item-type>
+		<label>EHUB, WLoadConsq L2</label>
```

Comment:
What does `WLoadConsq` mean? Is it possible to have give it a more meaningfull and easy to read (short) label?

Please check others too.

Reply:
WLoadConsq and all the others are the name of the parameter I have found in the specifications of the system. I think it will be best to use them. For those who has an description in the spec. are they as: <description> in channels.xml